### DISCUSSION TITLE
Is it possible to disable `.graphqlconfig` integration?

### DISCUSSION THREAD

**Discussion by Grohden** [graphql-hive/graphql-eslint#1685] [Q&A]
So, today I started to get this error:

```
➜  project git:(refactor/components-new-folders) ✗ yarn lint

Oops! Something went wrong! :(

ESLint: 8.16.0

[ProjectNotFoundError: File '/Users/gabrielrohden/Desktop/Projects/Company/project/app/javascript/packs/page_specific/somefile.js' doesn't match any project] {
  name: 'ProjectNotFoundError',
  message: "File '/Users/gabrielrohden/Desktop/Projects/Company/project/app/javascript/packs/page_specific/somefile.js' doesn't match any project"
```

It took me around 4 hours to discover that, yesterday, I changed my `.graphqlconfig` file to use the `includes` options so my IDE could provide better integration with the client schema

This change made eslint throw that error for any js file outside of the specific glob pattern match I assigned in the includes in the `.graphqlconfig`

so I wonder: is it possible to disable this `.graphqlconfig` auto integration with the plugin?

edit:

I also have the `skipGraphQLConfig: true` in the parser options, my overrides looks like this

```js
overrides: [
    {
      files: ['*.gql', '*.graphql'],
      parser: '@graphql-eslint/eslint-plugin',
      plugins: ['@graphql-eslint'],
      parserOptions: {
        skipGraphQLConfig: true,
        schema: './tmp/graphql/schema.json',
        operations: ['./app/javascript/**/*.{ts,tsx,js,jsx}'],
      },
      rules: {
        '@graphql-eslint/require-id-when-available': 'error',
        '@graphql-eslint/no-scalar-result-type-on-mutation': 'error',

        // we probably need to update prettier for it to work with
        // graphql-eslint properly
        'prettier/prettier': 'off',
      },
    },
    {
      files: ['*.js', '*.tsx', '*.ts'],
      processor: '@graphql-eslint/graphql',
      parserOptions: {
        skipGraphQLConfig: true,
      },
    },
]
```

but it seems to take no effect

**Comment by dimaMachina** [comment]
`skipGraphQLConfig: true` disables search physical graphql-config, if it doesn't work for you - provide a reproduction repo and I'll take a look at why it happens for you

**Comment by Grohden** [comment]
@B2o5T thx for answering!

here it is https://github.com/Grohden/graphql-eslint-issue-discussion-1685

let me know if I'm doing something wrong or what I can help with

**Reply by dimaMachina** [reply]
I receive follow error

```sh
/Users/dmytro/Desktop/repros/graphql-eslint-issue-discussion-1685/src/packs/unrelated-file.js
  0:0  error  Preprocessing error: File '/Users/dmytro/Desktop/repros/graphql-eslint-issue-discussion-1685/src/packs/unrelated-file.js' doesn't match any project
```

https://github.com/Grohden/graphql-eslint-issue-discussion-1685/blob/022c2f26f607ca7271813b039d944a35798ce876/.eslintrc.js#L45

you need to modify this line to `files: ['./src/domain/**/*.{ts,tsx,js,jsx}'],` as per your `parserOptions.operations` or did I misunderstand something?

**Reply by dimaMachina** [reply]
otherwise, what's the reason to extract documents from files that you didn't specify in graphql-config?

**Reply by Grohden** [reply]
hmm.. right, but it starts to fail the moment I'm using includes in the `.graphqlconfig`, shouldn't that be ignored due to the `skipGraphQLConfig`?

**Reply by dimaMachina** [reply]
could you push a commit with `includes`?

**Reply by Grohden** [reply]
https://github.com/Grohden/graphql-eslint-issue-discussion-1685/blob/022c2f26f607ca7271813b039d944a35798ce876/.graphqlconfig#L4 its here, if I remove this line it works again

**Reply by Grohden** [reply]
>but it starts to fail the moment I'm using includes in the .graphqlconfig

I mean, with the current state of the repo, without that fix you pointed out
